### PR TITLE
Showing Warning on Reset to Default button in App bar buttons

### LINKED ISSUE #13174: Reset to Default button in App bar button doesn't show warning..
**Is your feature request related to a problem? Please describe. **
App bar buttons doesn't show warning when "Reset to default" is clicked.

**Describe the solution you'd like**
Dialog showing "Do you want to proceed. Yes/No"

**Describe alternatives you've considered**
enabling undo button to restore previous settings.

**Additional context**
i would like to get permission to work on this issue.

https://user-images.githubusercontent.com/96579549/215783706-70ee8129-abff-47ec-af80-101b958d6b7b.mp4

### PR DESCRIPTION
**Pull request by SanjaySargam**
## Fixes
Fixes #13174 

## Screen Recording

[default.webm](https://user-images.githubusercontent.com/65113071/227170029-4f54d37e-7dce-4410-a508-5b5e4030a5ea.webm)

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by BrayanDSO** [comment]
lint

**Comment by SanjaySargam** [comment]
> lint

Now all tests passed

File: AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt [hunk 1]
```diff
@@ -34,30 +36,38 @@ class CustomButtonsSettingsFragment : SettingsFragment() {
...
+                            remove("customButtonEnableWhiteboard")
+                            remove("customButtonSaveWhiteboard")
+                            remove("customButtonWhiteboardPenColor")
+                            remove("customButtonClearWhiteboard")
+                            remove("customButtonShowHideWhiteboard")
+                        }
+                        // #9263: refresh the screen to display the changes
+                        refreshScreen()
+                    }
+                    .negativeButton(R.string.dialog_cancel, null)
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/preferences/CustomButtonsSettingsFragment.kt:69]:
```suggestion
                message(R.string.reset_confirmation_text)
                positiveButton(R.string.reset) {
                    // Reset the settings to default
                    AnkiDroidApp.getSharedPrefs(requireContext()).edit {
                        remove("customButtonUndo")
                        remove("customButtonScheduleCard")
                        remove("customButtonEditCard")
                        remove("customButtonTags")
                        remove("customButtonAddCard")
                        remove("customButtonReplay")
                        remove("customButtonCardInfo")
                        remove("customButtonSelectTts")
                        remove("customButtonDeckOptions")
                        remove("customButtonMarkCard")
                        remove("customButtonToggleMicToolBar")
                        remove("customButtonBury")
                        remove("customButtonSuspend")
                        remove("customButtonFlag")
                        remove("customButtonDelete")
                        remove("customButtonEnableWhiteboard")
                        remove("customButtonSaveWhiteboard")
                        remove("customButtonWhiteboardPenColor")
                        remove("customButtonClearWhiteboard")
                        remove("customButtonShowHideWhiteboard")
                    }
                    // #9263: refresh the screen to display the changes
                    refreshScreen()
                }
                negativeButton(R.string.dialog_cancel, null)
```

File: AnkiDroid/src/main/res/values/10-preferences.xml [hunk 2]
```diff
@@ -178,6 +178,9 @@
     <string name="custom_buttons_setting_if_room">Show if room</string>
     <string name="custom_buttons_setting_menu_only">Menu only</string>
     <string name="reset_custom_buttons" maxLength="41">Reset to default</string>
+    <string name="reset_settings_to_default">Reset Settings to Default</string>
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values/10-preferences.xml:181]: Phrase casing
```suggestion
    <string name="reset_settings_to_default">Reset settings to default</string>
```

File: AnkiDroid/src/main/res/values/10-preferences.xml [hunk 3]
```diff
@@ -178,6 +178,9 @@
...
+    <string name="reset_confirmation_text" maxLength="41">Are you sure you want to reset your settings to default?</string>
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/res/values/10-preferences.xml:182]: *Particularly*, I'd prefer to remove this message, as the dialog title is pretty clear already

**Review by BrayanDSO — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/res/values/10-preferences.xml (hunk 3 above)
Comment (SanjaySargam) [reply AnkiDroid/src/main/res/values/10-preferences.xml:182]: Ok

**Review by BrayanDSO — APPROVED** [review]

**Comment by mikehardy** [comment]
I'm okay with the status of strings right now based on crowdin activity stream. 1 Japanese change + 11 German changes which I can review in crowdin API, so, merging this one

**Pull request merged**